### PR TITLE
Post/hello world

### PR DESCRIPTION
**Pull request by barbosa**

### CODE REVIEW

File: content/posts/2020-06-01-hello-world/index.md [hunk 1]
```diff
@@ -1,50 +1,22 @@
...
 lang: en
-path: /blog/hello-world/
-date: '2020-06-01'
-tags: ['blogs']
+path: /blog/hello-world
+date: "2020-06-01"
 ---
 
-Hello World from the other side.
+In 2014, Loadsmart was born. At that time, we had lots of ideas in mind about what we wanted to achieve and about everything we were planning to build to solve the challenges many companies face in the logistics industry. We knew the impact could be big. In a field where so many processes are still to the present date made by repetitive tasks, phone calls, and an extensive list of touch points, technology is the only option to scale efficiently.
```
Comment (aarondroseman) [inline content/posts/2020-06-01-hello-world/index.md:11]: At that time, we had lots of ideas about how we wanted to achieve everything we were planning to build to solve the many challenges logistics companies face.

Comment (aarondroseman) [inline content/posts/2020-06-01-hello-world/index.md:11]: Change made - > completed in 4th sentence

File: content/posts/2020-06-01-hello-world/index.md [hunk 2]
```diff
@@ -1,50 +1,22 @@
...
 
-```javascript
-parseInt(string, base);
-```
+We were a small yet excited team sharing a tiny little office in New York City, in one of the busiest avenues in Midtown Manhattan. We would bump each others' chairs a couple of times a day when coming back to our desks after a meeting or an always pleasant team lunch (we miss [Morocho](https://www.yelp.com/biz/morocho-peruvian-fusion-new-york) so much!).
```
Comment (aarondroseman) [inline content/posts/2020-06-01-hello-world/index.md:13]: ON one of the busiest avenues in Midtown Manhattan..

File: content/posts/2020-06-01-hello-world/index.md [hunk 3]
```diff
@@ -1,50 +1,22 @@
...
 
-## Examples
+Well, many things changed since then. We grew from a team of 4 engineers in 2015 to more than 70 in 2020. We switched our tech stack (a couple of times), cloud providers, processes. We learned **tons** on this journey. And in the end, this is the reason we're launching this blog. To share some of our tech challenges, projects, and all the cool things we have been building so far.
```
Comment (aarondroseman) [inline content/posts/2020-06-01-hello-world/index.md:15]: Well, many things have changed since then.

Comment (aarondroseman) [inline content/posts/2020-06-01-hello-world/index.md:15]: ...and all the cool things we have built so far.

**Comment by davidsonfellipe** [comment]
@barbosa I didn't see any change on package.json, but this PR is updating the `yarn.lock`, did you add any new dependency?

**Pull request merged**